dalle add model parameter

AzeWZ opened the pull request:
- **Description:** dalle_image_generator adding a new model parameter,
  - **Issue:** N/A,
  - **Dependencies:** 
  - **Tag maintainer: @hwchase17
  - **Twitter handle:**

Commit:
dalle add model parameter

Commit:
"format"

eyurtsev commented:
cc @matthewdeguzman

eyurtsev committed:
Merge branch 'master' into dalle_add_model_parameter

eyurtsev committed:
x

AzeWZ committed:
Merge branch 'master' into dalle_add_model_parameter

AzeWZ committed:
Merge branch 'master' into dalle_add_model_parameter

efriis commented on libs/langchain/langchain/utilities/dalle_image_generator.py:
```diff
@@ -21,6 +24,8 @@ class DallEAPIWrapper(BaseModel):
     """Size of image to generate"""
     separator: str = "\n"
     """Separator to use when multiple URLs are returned."""
+    model: str = "dall-e-3"
```
should we remove the default to have it automatically use OpenAI's default (currently dalle 2), to not have it auto-upgrade people unexpectedly?
```suggestion
    model: Optional[str] = None
```

eyurtsev replied:
Yeah the default should match the underlying API

AzeWZ committed:
Update libs/langchain/langchain/utilities/dalle_image_generator.py

remove the default to have it automatically use OpenAI's default (currently dalle 2)

Co-authored-by: Erick Friis <erickfriis@gmail.com>

AzeWZ committed:
Merge branch 'master' into dalle_add_model_parameter

AzeWZ committed:
Merge branch 'langchain-ai:master' into dalle_add_model_parameter
